build/docker/zap-baseline.py

```diff
@@ -53,10 +53,12 @@
 import sys
 import time
 import traceback
-import urllib2
+from six.moves.urllib.parse import urlparse, urlencode
+from six.moves.urllib.request import urlopen
+from six.moves.urllib.error import HTTPError
 from datetime import datetime
 from random import randint
-from zapv2 import ZAPv2
+#from zapv2 import ZAPv2
```

Comment:
@thc202 thank you so much for the review. It was my mistake. I forgot to uncomment that line.